tests/unit/Administration/Framework/Routing/KnownIps/KnownIpsCollectorTest.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Administration\Framework\Routing\KnownIps;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Administration\Framework\Routing\KnownIps\KnownIpsCollector;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @internal
+ */
+final class KnownIpsCollectorTest extends TestCase
+{
+    public function testDeleteWillRemoveUserConfigs(): void
```

Comment:
can you fix the method name, I think you copied it but didn't changed the method name

Reply:
Maybe :DDD ty for noticing this